index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portofolio</title>
+  </head>
+  <body>
+    <!--The main container: all elements are in-->
+    <main id="id-main-content">
+      <!--the document header-->
+      <header id="id-header-content">
+        <img src="/images/ic_menu.svg" alt="menu button" id="id-side-menu">
+      </header>
+
+      <!--this section contain information about the author-->
+      <section id="id-main-section">
+        <p id="id-my-name">Marc Raphael</p>
+
+        <!--Information about the author-->
+        <p id="id-about-me">
+          I am a software developer! i can help you build a product, feature or website.
+          look through some of my work and experience! If you like what you see and have
+          a project you need coded, feel free to contact me!
+        </p>
+
+        <!--Here is the transition to the next section-->
+        <div id="id-start-see-project">
+          <span id="id-title-see-project">Start collaboration</span>
+        </div>
```

Comment:
This element is a button element. Don't you think a button- or a- tag is appropriate? Kindly apply the requested change.

Reply:
it is looking like a header. but you are right!